mkosi/backend.py

```diff
@@ -141,33 +141,34 @@ def __str__(self) -> str:
 class Distribution(enum.Enum):
     package_type: PackageType
 
-    fedora = 0, PackageType.rpm
-    debian = 1, PackageType.deb
-    ubuntu = 2, PackageType.deb
-    arch = 3, PackageType.pkg
-    opensuse = 4, PackageType.rpm
-    mageia = 5, PackageType.rpm
-    centos = 6, PackageType.rpm
-    centos_epel = 7, PackageType.rpm
-    openmandriva = 10, PackageType.rpm
-    rocky = 11, PackageType.rpm
-    rocky_epel = 12, PackageType.rpm
-    alma = 13, PackageType.rpm
-    alma_epel = 14, PackageType.rpm
-    gentoo = 15, PackageType.ebuild
-
-    def __new__(cls, number: int, package_type: PackageType) -> Distribution:
+    fedora = "fedora", PackageType.rpm
+    debian = "debian", PackageType.deb
+    ubuntu = "ubuntu", PackageType.deb
+    arch = "arch", PackageType.pkg
+    opensuse = "opensuse", PackageType.rpm
+    mageia = "mageia", PackageType.rpm
+    centos = "centos", PackageType.rpm
+    centos_epel = "centos_epel", PackageType.rpm
+    openmandriva = "openmandriva", PackageType.rpm
+    rocky = "rocky", PackageType.rpm
+    rocky_epel = "rocky_epel", PackageType.rpm
+    alma = "alma", PackageType.rpm
+    alma_epel = "alma_epel", PackageType.rpm
+    gentoo = "gentoo", PackageType.ebuild
```

Comment:
Hmm, would it be possible to align the names at the first `"`? Without that, it's really hard to scan the list. And also the second item column too…